setup.py

```diff
@@ -37,13 +37,20 @@ def get_version():
     return version_file.read().strip()
 
 
+def read_readme():
+    with open('README.md', 'r') as f:
+        return f.read()
+
+
 VERSION = get_version()
 
 
 setup(
     name='vim-vint',
     version=VERSION,
     description='Lint tool for Vim script Language',
+    long_description=read_readme(),
+    long_description_content_type="text/markdown",
```

Comment:
It's nice that markdown is supported by now, but why do you convert the existing docs?
Do you prefer Markdown?
(just curious)

Reply:
In my memory, I did not know that what markup languages can be used as `long_description`. But I found an official example that using with `long_description_content_type="text/markdown"`, and I was not sure about what MIME types can be represented as reStructuredText (probably it is [`text/x-rst`](http://docutils.sourceforge.net/FAQ.html#what-s-the-official-mime-type-for-restructuredtext-data) but I could not be sure). So I decided to use Markdown.